package.json

```diff
@@ -14,22 +14,22 @@
   },
   "homepage": "https://github.com/scality/vaultclient#readme",
   "dependencies": {
-    "@aws-crypto/sha256-universal": "^2.0.1",
-    "@smithy/signature-v4": "^2.1.1",
-    "commander": "2.20.0",
-    "httpagent": "git+https://github.com/scality/httpagent#1.0.6",
-    "werelogs": "scality/werelogs#8.1.3",
-    "xml2js": "0.4.19"
+    "@aws-crypto/sha256-universal": "^5.2.0",
+    "@smithy/signature-v4": "^4.1.0",
+    "commander": "^11.0.0",
+    "httpagent": "git+https://github.com/scality/httpagent#5acb093635060ec8289c891e2d525535bef93bec",
+    "werelogs": "scality/werelogs#0ab4ca3470a2f03f34d33fdc2689fda414f7c206",
+    "xml2js": "^0.6.2"
   },
   "devDependencies": {
-    "arsenal": "git+https://github.com/scality/Arsenal#8.1.91",
-    "babel-eslint": "10.0.2",
-    "eslint": "6.8.0",
-    "eslint-config-airbnb": "17.1.0",
-    "eslint-config-scality": "scality/Guidelines#8.2.0",
-    "eslint-plugin-import": "^2.18.0",
-    "mocha": "6.1.4",
-    "typescript": "^4.7.3"
+    "@babel/eslint-parser": "^7.25.1",
+    "arsenal": "git+https://github.com/scality/Arsenal#8.1.135",
+    "eslint": "8.57.0",
```

Comment:
can we switch to eslint 9 ?

to simplify upgrade of eslint, can we/shoudl we migrate and upgrade `scality/Guidelines` ? (and remove custom config from repos, ideally...)

Reply:
It doesn't work well today to create a new eslintv9 module to serve as a new config, I tried for 3h+ without success between last week and this week: there is a sub dependency that is not yet compatible, and it makes things fail, I was not able to make CTST with the updated scality/guidelines work with eslint v9.
The issue is https://github.com/import-js/eslint-plugin-import/issues/2948 (but there are others, because not all the configs are upgraded yet). The module from the link is not optional as far as I know